TCAT/Controllers/HomeViewController.swift

```diff
@@ -132,11 +133,14 @@ class HomeViewController: UIViewController {
 
     @objc func reachabilityChanged(_ notification: Notification) {
         if let reachability = notification.object as? Reachability {
-            
+
             // Dismiss current banner, if any
             banner?.dismiss()
             banner = nil
-            
+            // Dismiss current loading indicator, if any
```

Comment:
thoughts on turning the `if let` into a `guard let reachability = notification.object as? Reachability else { return }`

This will then get rid of all the code indented one block. IMO this is cleaner